kf_lib_data_ingest/etl/configuration/log.py

```diff
@@ -10,6 +11,26 @@
 )
 
 
+def _log_level_dict():
+    """
+    Create log levels dict, where a key is the string representation of
+    log level and a value is the int representation of the level.
+
+    Log levels dict is used in cli.py to provide the enumeration of acceptable
+    choices for --log_level param.
+    """
+    # Log levels dict from logging
+    log_levels = logging._levelToName
+    # Remove the `noset` level, we don't need that
+    log_levels.pop(0)
+
+    return {v.lower(): k for k, v in log_levels.items()}
```

Comment:
using .lower() here just means that you also have to use .lower() down in setup_logger.